x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -240,6 +250,59 @@ export class TelemetryReceiver {
     };
   }
 
+  public async fetchDetectionRules() {
+    if (this.esClient === undefined || this.esClient === null) {
+      throw Error('elasticsearch client is unavailable: cannot retrieve diagnostic alerts');
+    }
+
+    const query: SearchRequest = {
+      expand_wildcards: 'open,hidden',
+      index: '.kibana*',
```

Comment:
hmm, this is going to need to change to the users default index, and the size below should be constant. Will patch.